tests/jest.config.dom.js

```diff
@@ -1,7 +1,12 @@
 const path = require('path')
-const baseConfig = require('kcd-scripts/jest')
+const {
+  // global config options that would trigger warnings in project configs
+  collectCoverageFrom,
+  watchPlugins,
+  ...baseConfig
+} = require('kcd-scripts/jest')
```

Comment:
@kentcdodds Would it make sense to provide `kcd-scripts/jest-global` and `kcd-scripts/jest-project` directly so that consumers of `kcd-scripts` don't have to manually cherry-pick?

Reply:
That's fine for me. Happy to merge a PR for that.